lib/SmileIdentityCore.php

```diff
@@ -215,23 +215,25 @@ public function get_job_status($partner_params, $options)
 
     /***
      *  Will query the backend for web session token with a specific timestamp
-     * @param timestamp the timestamp to generate the token from
+     * @param timestamp the iso 8601 date/time format to generate the token from
      * @param user_id
      * @param job_id
      * @param product_type - Literal value of any of the 6 product type options
      * @return array
      * @throws GuzzleException
      */
-    public function get_web_token($timestamp, $user_id, $job_id, $product_type): array
+    public function get_web_token($user_id, $job_id, $product_type, $timestamp = null, $callback_url = null): array
```

Comment:
Any reason for swapping arguments/parameters?

Reply:
Ope asks we swap it so the method signature is similar with the other libs

Reply:
Okay

Reply:
if I recall correctly, the `get_token` method does not use a timestamp.

also, the callback_url is not optional. please adjust the function, and the comments.